scapy/layers/dns.py

```diff
@@ -19,12 +19,148 @@
 from scapy.sendrecv import sr1
 from scapy.layers.inet import IP, DestIPField, UDP, TCP
 from scapy.layers.inet6 import DestIP6Field
-from scapy.error import warning
+from scapy.error import warning, Scapy_Exception
 from functools import reduce
 import scapy.modules.six as six
 from scapy.modules.six.moves import range
 
 
+def DNSgetstr(s, p, pkt=None, _internal=False):
```

Comment:
Could you rename the `DNSxxx` functions to `dns_xxx`?

Reply:
Of course. Let’s just hope that no one was using DNSgetstr (it wasn’t documented anyways).

I kept the original format

Reply:
I think you can safely rename `DNSgetstr`. If you want, you can add a `DNSgetstr` function that calls the new one and add a deprecation warning.